cli/src/lib.rs

```diff
@@ -284,6 +301,7 @@ where
             kura,
             block_sync,
             torii,
+            thread_handlers: vec![kura_thread_handler],
```

Comment:
So what do we need `Vec` for? For a future?

Reply:
Yes. There will be more threads in the future, such as the `sumeragi_thread` that exists on `iroha2-staging`.